### ISSUE TITLE
Fix `install_opencv` in the RoSys docker image to support omnidirectional cameras

### ISSUE THREAD

**Issue by codingpaula** [zauberzeug/rosys#182]
Currently the docker image doesn't install the opencv requirements like in the pytest action and as specified in the `install_opencv.py` script. But the dependencies in the script are necessary to e.g. run the tests correctly. When adding the same line as in the pytest action to the Dockerfile, I get the following error:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'install_opencv'
```

**Comment by rodja** [comment]
@codingpaula is that fixed with  #181?

**Comment by codingpaula** [comment]
No, #181 just fixes the base image and installing the full project (before running the image resulted in missing the version).

**Comment by falkoschindler** [comment]
In b43b47c0780900f5888aa093293723310013db1f I added checks if `omnidir` is properly installed. Otherwise we simply ignore corresponding tests. This allows running tests locally and in other environments without the special `install_opencv` workaround. So this issue can focus on making camera calibration for omnidirectional cameras available to user code based on the RoSys docker image.

**Comment by NiklasNeugebauer** [comment]
We have now fixed OpenCV to `4.9.0.80`, also set the contrib module in the pyproject.
This limits numpy to < `2.0` since OpenCV < `4.10` has only been compiled with `1.X`.
We will update OpenCV as soon as it supports the omnidir module again.

**Comment by NiklasNeugebauer** [comment]
We have now fixed OpenCV to `4.9.0.80` and set the contrib module in the pyproject.
This limits numpy to < `2.0` since OpenCV < `4.10` has only been compiled with `1.X`.
We will update OpenCV as soon as it supports the omnidir module again.

This fixes the issue and omnidir calibration is tested again.
